UIKitBasics/UIKitBasics/ProfileViewController.swift

```diff
@@ -0,0 +1,36 @@
+import UIKit
+
+
+class ProfileViewController: UIViewController {
+
+    @IBOutlet var greetingLabel: UILabel!
+    @IBOutlet var signOutButton: UIButton!
+    var username: String = ""
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        greetingLabel.text = "Hello, \(username)"
+        setConstraints()
+    }
+    
+    @IBAction func signOutButtonDidTap(_ sender: Any) {
+        self.dismiss(animated: true, completion: nil)
```

Comment:
Либо не писать self, либо вообще вот так:
```suggestion
        presentingViewController?.dismiss(animated: true, completion: nil)
```